scipy/stats/_stats_py.py

```diff
@@ -4363,6 +4366,10 @@ def confidence_interval(self, confidence_level=0.95, method=None):
         return ci
 
 
+xp_capabilities_table['pearsonr'] = dict(cpu_only=True)
+
+
+@xp_capabilities(xp_capabilities_table)
```

Comment:
The job of the decorator that gets applied to the function is to add the table to the documentation. The decorator factory `xp_capabilities` accepts the module's `xp_capabilities_table` as an argument. The decorator returned by the factory gets passed the function as an argument `f`, so inside that we can look up `xp_capabilities_table[f.__name__]`.

Reply:
There was some concern when I originally suggested the idea that using decorators to inject the docs might cause too much of a slowdown at import time if there are lots of functions. However it seems reasonable to me to start with something like this and then, if this becomes an issue, move to some sphinx magic

Reply:
Yeah. But if that comes up again, then it's basically saying that only rendered docs matter. If that's the case, then we should be "allowed" to use :math: liberally.